src/test/java/GmailActionableEmailsServletTest.java

```diff
@@ -46,10 +48,12 @@
   private static final List<String> SUBJECT_LINE_PHRASES_LIST =
       Arrays.asList("Action Word One", "ActionWordTwo");
 
-  private static final List<String> METADATA_HEADERS = ImmutableList.of("Subject", "From");
+  private static final List<String> METADATA_HEADERS = ImmutableList.of("Subject", "From", "To");
 
-  private static final String MESSAGE_ID_ONE = "messageOne";
-  private static final String MESSAGE_ID_TWO = "messageTwo";
+  private static final String MESSAGE_ID_OLD = "messageOne";
+  private static final String MESSAGE_ID_NEW = "messageTwo";
+  private static final String MESSAGE_ID_LOW_PRIORITY = "messageTwo";
+  private static final String MESSAGE_ID_HIGH_PRIORITY = "messageTwo";
   private static final String SUBJECT_VALUE_ONE = "subjectValueOne";
   private static final String SUBJECT_VALUE_TWO = "subjectValueTwo";
   private static final String SENDER_EMAIl = "example@example.com";
```

Comment:
Nit: the last letter should be capital 'L'

Reply:
Fixed! Good catch